Example/AppDelegate.swift

```diff
@@ -56,7 +56,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         _ = try? FileManager.default.createDirectory(at: historyRecordingUrl,
                                                      withIntermediateDirectories: true,
                                                      attributes: nil)
-
+        
+        let settingsValues = NavigationSettings.Values(logLevel: .debug)
+        NavigationSettings.shared.initialize(with: settingsValues)
```

Comment:
Let's keep the default log level in the sample app to allow the clients to see the default SDK behavior